### DISCUSSION TITLE
Dropping support for Visual C++ 2015 Update 3

### DISCUSSION THREAD

**Discussion by trueqbit** [microsoft/vcpkg#29149] [General]
This may not be true for everyone, but it seems to me that the world is moving towards C++14 as a minimal baseline.
Important libraries like Catch2 v3 that [made it into the vcpkg ecosystem](https://github.com/microsoft/vcpkg/pull/24762) now require a fully compliant C++14 compiler, and maintainers are [not interested in maintaining compatibility](https://github.com/catchorg/Catch2/issues/2624#issuecomment-1400979715), and rightly so.

There was a [long discussion](https://github.com/microsoft/vcpkg/pull/24762) about some ports that depend on Catch2 v2, and whether to patch them. Although I'm a little late to the party, I'd still like to mention that it's not just about the dependency of [sqlite-orm] and other ports on an old version of Catch2, but also on the base C++ standard.

In the prerequisites for Windows, Visual Studio 2015 Update 3 is listed.
The problem with this compiler is that it is lacking

- Extended constexpr
- Aggregates with standard initialisations for members
- Adequate support for template aliasing.

On that note - what is the policy of vcpkg? I vote to move on and raise the baseline to C++14.

**Comment by autoantwort** [comment]
There is not really a baseline for vcpkg ports, only for the tool.

**Comment by JackBoosY** [comment]
I vote too.
VS 2015 is too old that we can hardly take a look at the issues.
